docat/docat/app.py

```diff
@@ -113,6 +113,44 @@ def get_project(project):
     )
 
 
+@app.post("/api/{project}/icon", response_model=ApiResponse, status_code=status.HTTP_200_OK)
+@app.post("/api/{project}/icon/", response_model=ApiResponse, status_code=status.HTTP_200_OK)
+def upload_icon(
+    project: str,
+    response: Response,
+    file: UploadFile = File(...),
+    docat_api_key: Optional[str] = Header(None),
+    db: TinyDB = Depends(get_db),
+):
+    project_base_path = DOCAT_UPLOAD_FOLDER / project
+    icon_path = project_base_path / "logo"
+
+    if not project_base_path.exists():
+        response.status_code = status.HTTP_404_NOT_FOUND
+        return ApiResponse(message=f"Project {project} not found")
+
+    if not file.content_type.startswith("image/"):
```

Comment:
Me uploading the first gif icon:

![cat-lover-awesome](https://user-images.githubusercontent.com/5184499/191557233-9551f611-0495-410d-ba2b-14788dd56bcf.gif)